common/modal.js

```diff
@@ -17,20 +17,27 @@
             $uibModalInstance.dismiss('cancel');
         }
     }])
-    .controller('ErrorModalController', ['$uibModalInstance', 'params', 'messageMap', function ErrorModalController($uibModalInstance, params, messageMap) {
+    .controller('ErrorModalController', ['$uibModalInstance', 'params', 'messageMap', '$window', function ErrorModalController($uibModalInstance, params, messageMap, $window) {
         var vm = this;
         vm.params = params;
         vm.details = false;
```

Comment:
Could you also change this and corresponding calling sites to `showDetails`.

Reply:
`showDetails` was already being used. Changed it to `displayDetails`.